packages/kit/types/config.d.ts

```diff
@@ -44,6 +44,12 @@ export type Config = {
 		host?: string;
 		hostHeader?: string;
 		hydrate?: boolean;
+		serviceWorker?: {
+			// Glob patterns relative to `files.assets` dir. Files matching this would not be available in $service-worker.files
+			// e.g. if `files.assets` has value `static` then ['og-tags-images/**/*'] would match all files under `static/og-tags-images` dir.
+			// As og-tags-images are never loaded by a normal client(browser), they can be comfortably skipped from here.
```

Comment:
These lines doesn't belong here

Reply:
@ignatiusmb Hey. What would be a better place then?
I added it here because when using an IDE(as a user of SvelteKit) you go to the definition of `serviceWorker.exclude`, it would immediately let you know what this option is for. 

I have removed the comment from there as per your suggestion but would like to know where the comment should go.

Reply:
We usually just put it in the docs to keep it maintainable and that should be enough for most cases. I do think we can improve these hover/definitions, but we'll need to do it for all of them and not just this specific one.

Reply:
Cool. Thanks.